widgets/src/components/common/Table/__tests__/LazyTablePagination.test.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable react/destructuring-assignment */
```

Comment:
still needed? not sure which part is throwing an error

Reply:
removed!